manager/volume.go

```diff
@@ -501,6 +499,61 @@ func (m *VolumeManager) Activate(volumeName string, frontend string) (v *longhor
 	return v, nil
 }
 
+func (m *VolumeManager) syncBackupAndRestoreWithLastBackup(volume *longhorn.Volume, engine *longhorn.Engine) (bool, error) {
+	// trigger BackupVolume synchronization
+	var backupVolumeName string
+	if volume.Status.RestoreRequired {
+		name, ok := volume.Labels[types.LonghornLabelBackupVolume]
+		if !ok {
+			return false, errors.New("Cannot find the backup volume label")
```

Comment:
Add `volume.Name` to the error message and make the first word lowercase "cannot ....".

Reply:
Got it. I will fix the error to with adding volume name and lowercase the sentence.